charts/internallb-webhook-admission-controller/templates/internallb-webhook-admission-controller-deployment.yaml

```diff
@@ -26,6 +26,13 @@ spec:
       - name: webhook
         image: {{ .Values.controller.image }}:{{ .Values.controller.imageTag }}
         imagePullPolicy: {{ .Values.controller.imagePullPolicy }}
+        command: ["/internallb-webhook-admission-controller"]
+        args:
+        - --svcannotationkey={{ .Values.controller.serviceAnnotationKey }}
+        - --svcannotationvalue={{ .Values.controller.serviceAnnotationValue }}
+        - --alsologtostderr
```

Comment:
You'll see in #13 that I've changed this argument to simply `logtostderr` as there's no `/tmp` for glog to write its file to when the image is `FROM scratch`. If you take that, depending on merge order one PR or the other will need to be updated.

Reply:
Yeah that's fine